src/dc_receive_imf.rs

```diff
@@ -850,10 +840,20 @@ fn save_locations(
     insert_msg_id: MsgId,
     hidden: i32,
 ) {
+    if chat_id <= DC_CHAT_ID_LAST_SPECIAL as libc::c_uint {
```

Comment:
libc conversion is not necessary, constant is already `u32` typed, same as `chat_id`.